manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer-windows.yaml

```diff
@@ -16,6 +16,9 @@ spec:
 {{ include "sscdpa.labels" . | indent 6 }}
 {{- if .Values.windows.podLabels }}
 {{- toYaml .Values.windows.podLabels | nindent 8 }}
+{{- end }}
+{{- if .Values.linux.podAnnotations }}
+{{- toYaml .Values.linux.podAnnotations | nindent 8 }}
```

Comment:
```
{{- if .Values.windows.podAnnotations }}
  annotations:
{{- toYaml .Values.windows.podAnnotations | nindent 8 }}
{{- end }}
```

Reply:
The values here should be `.Values.windows`

Reply:
Fixed this as well. I'm sorry but I'm a little tired today.